chroma-manager/tests/integration/existing_filesystem_configuration/utils/create_lustre_filesystem.py

```diff
@@ -191,6 +194,12 @@ def create_lustre_filesystem_for_test(self):
 
         self._save_modified_config()
 
+        for server in config['lustre_servers']:
```

Comment:
If we don't do this, are we not starting the agents?

Reply:
It seemed that we would want to do it after setting up the server since it gets disabled before the restart. Would there be another place that does this?

Reply:
I think I may have thought this was an oversight, but if the agent isn't explicitly started already, it may have been by design.

Reply:
Ok I will remove this line then :+1